### DISCUSSION TITLE
Flux Lora merge?

### DISCUSSION THREAD

**Discussion by PheonixAi420** [hako-mikan/sd-webui-supermerger#400] [Q&A]
Has anyone tried merging Flux Loras in Supermerger yet? I tried a few times but kept getting "block weight errors" Not sure what I am doing wrong. Anyone know how to do it?

**Comment by TomLucidor** [comment]
Still a bit new, they need their own adaptor to handle weights... also keep an eye on AuraFlow

**Comment by kalle07** [comment]
same question
